apps/mobile/package.json

```diff
@@ -18,13 +18,11 @@
 		"build": "react-native run-android --variant=release"
 	},
 	"dependencies": {
-		"@gorhom/bottom-sheet": "^2",
+		"@gorhom/bottom-sheet": "^4",
 		"@react-native-async-storage/async-storage": "^1.15.1",
 		"@react-native-community/masked-view": "^0.1.5",
 		"@react-native-community/netinfo": "^6.0.0",
 		"@react-native-community/slider": "^4.1.12",
-		"@react-native-firebase/analytics": "^15.2.0",

```

Comment:
<img width="1017" alt="Screenshot 2024-07-07 at 11 09 28 AM" src="https://github.com/YajanaRao/Serenity/assets/38914337/fc4a2fad-d067-4cf1-80fc-e63decfdf755">

Presence of firebase package is causing in android build failures.